opm/input/eclipse/Schedule/Well/WellEnums.cpp

```diff
@@ -27,6 +27,15 @@
 
 #include <fmt/format.h>
 
+template<>
+struct fmt::formatter<Opm::WellWELTARGCMode> : fmt::formatter<int>
+{
+  auto format(const Opm::WellWELTARGCMode& cmode, format_context& context) const
```

Comment:
sorry missed it. and indent 4

Reply:
> indent 4

Right&ndash;sorry about that.  I've pushed an update to fix that now.